protos/proto/aptos/indexer/v1/grpc.proto

```diff
@@ -0,0 +1,108 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package aptos.indexer.v1;
+
+import "aptos/indexer/v1/raw_data.proto";
+import "aptos/transaction/v1/transaction.proto";
+import "aptos/util/timestamp/timestamp.proto";
+
+message StreamProgressSampleProto {
+  optional aptos.util.timestamp.Timestamp timestamp = 1;
+  uint64 version = 2;
+  uint64 size_bytes = 3;
+}
+
+message StreamProgress {
+  repeated StreamProgressSampleProto samples = 1;
```

Comment:
What's the intent behind collecting multiple samples here?

Reply:
calculate tps, etc.

Reply:
sync'ed offline; for monitoring purpose.